util/opentelemetry-util-genai/tests/test_upload.py

```diff
@@ -155,7 +156,7 @@ def test_upload_then_shutdown(self):
         )
         # all items should be consumed
         self.hook.shutdown()
-
+        time.sleep(2)
```

Comment:
Can you file an issue for the flake and leave a comment in code? I'll try to fix it in a follow up